src/navigation/Routes.android.js

```diff
@@ -300,6 +269,7 @@ function BSNavigator() {
       <BSStack.Screen
         component={AddWalletNavigator}
         name={Routes.ADD_WALLET_NAVIGATOR}
+        options={addWalletNavigatorPreset}
```

Comment:
this new preset allows a custom, dynamic sheet height to be provided to the navigator